lua/lang/zig.lua

```diff
@@ -1,13 +1,30 @@
 local M = {}
 
 M.config = function()
-  -- TODO: implement config for language
-  return "No config available!"
+  O.lang.zig = {
+    formatter = {
+      exe = "zig",
+      args = { "fmt" },
+      stdin = true,
+    },
+  }
 end
 
 M.format = function()
-  -- TODO: implement formatter for language
-  return "No formatter available!"
+  O.formatters.filetype["zig"] = {
+    function()
+      return {
+        exe = O.lang.zig.formatter.exe,
+        args = O.lang.zig.formatter.args,
+        stdin = not (O.lang.zig.formatter.stdin ~= nil),
```

Comment:
can you change this to ` stdin = O.lang.zig.formatter.stdin`
see #987 for more info